erpnext_documentation/www/docs/v13/user/manual/en/human-resources/employee-referral.md

```diff
@@ -0,0 +1,56 @@
+<!-- add-breadcrumbs -->
+
+# Employee Referral
+
+> Introduced In Version 13
+
+Internal Recruitment is one of the best processes for recruitment, and it also saves effort and capital.
+The Employee Referral is a process where existing employees refer a suitable candidate from their network for a vacant designation/position.
+
+In ERPNext, you can manage Employee Referrals.
+
+To access Employee Referral, go to:
+
+> Human Resources > Recruitment > Employee Referral
+
+
+## 1. Prerequisites
+1. [Employee](/docs/user/manual/en/human-resources/employee)
+1. [Additional Salary](/docs/user/manual/en/human-resources/additional-salary)
+1. [Job Applicant](/docs/user/manual/en/human-resources/job-applicant)
```

Comment:
Please refactor these links to point to the correct version (v13).

Reply:
Done